docs/prometheus.md

```diff
@@ -34,16 +34,22 @@ backend:
   #   Content-Type: application/json
   #   Authorization: Basic b2s6cGFzcW==
   measurement:
-    filter_good: prometheus_http_requests_total{code=~"2..", handler="/metrics"}[window]
-    filter_valid: prometheus_http_requests_total{handler="/metrics"}[window]
+    filter_good: http_requests_total{handler="/metrics", code=~"2.."}[window]
+    filter_valid: http_requests_total{handler="/metrics"}[window]
+    # operators: ['sum', 'rate']
 ```
 * The `window` placeholder is needed in the query and will be replaced by the
 corresponding `window` field set in each step of the Error Budget Policy.
 
 * The `headers` section (commented) allows to specify Basic Authentication
 credentials if needed.
 
-**&rightarrow; [Full SLO config](../samples/prometheus/slo_prom_metrics_availability_good_bad.yaml)**
+* The `operators` section defines which PromQL functions to apply on the
+timeseries. The default is to compute `sum(increase([METRIC_NAME][window]))` to
```

Comment:
The function `increase` should be only used for metrics of type `counter`:

> increase should only be used with counters. It is syntactic sugar for rate(v) multiplied by the number of seconds under the specified time range window, and should be used primarily for human readability. Use rate in recording rules so that increases are tracked consistently on a per-second basis.

(https://prometheus.io/docs/prometheus/latest/querying/functions/#increase)

Reply:
What other significant other metric types than counters do you think you would use for a `good_bad_ratio` method ? 
Histograms have their own method `distribution_cut`. 
For gauges, I'm not sure if measuring an SLO on them would make sense here, since most (all?) metrics we use for SLOs (throughput, # errors, # requests) are counters.

Reply:
An example would be a freshness SLO:

"95% of messages in queue "my-queue" are fresh"  (e.g. messages with age < 15m are considered as fresh).

```
  measurement:
    expression:    >
      count_over_time((max by (queue_name) (sqs_approximate_age_of_oldest_message_maximum{queue_name="my-queue"}) < 900)[window:])
      /
      count_over_time(max by (queue_name) (aws_sqs_approximate_age_of_oldest_message_maximum{queue_name="my-queue"})[window:])
```

Using operator `increase` would be incorrect in that case since it's a gauge.

Reply:
Okay, for `good_bad_ratio` those operators are just a default.

You can override the defaults by setting `operators` in your SLO config, could you please try this out:
```
  method: good_bad_ratio
  measurement:
    filter_good:    >
      (max by (queue_name)(sqs_approximate_age_of_oldest_message_maximum{queue_name="my-queue"}) < 900)[window:])
    filter_bad: >
      (max by (queue_name)(aws_sqs_approximate_age_of_oldest_message_maximum{queue_name="my-queue"})[window:])
    operators: [count_over_time]
```

Reply:
Thx. i will give a try and report back.